auth/auth_test.go

```diff
@@ -22,23 +22,6 @@ import (
 	ch "github.com/couchbase/sync_gateway/channels"
 )
 
-var gTestBucket base.Bucket
-
-func init() {
-	var err error
-
-	gTestBucket, err = base.GetBucket(base.BucketSpec{
-		Server:          base.UnitTestUrl(),
-		CouchbaseDriver: base.ChooseCouchbaseDriver(base.DataBucket),
-		BucketName:      "sync_gateway_tests"}, nil)
-	if err != nil {
-		log.Fatalf("Couldn't connect to bucket: %v", err)
-	}
-	if err != nil {
-		log.Fatalf("Couldn't install design doc: %v", err)
-	}
-}
```

Comment:
I'm very happy to see this code get pulled out - I've had it cause noise in a few different scenarios.